src/vm/builtins/builtin_runner/ec_op.zig

```diff
@@ -10,6 +11,14 @@ const MemorySegmentManager = @import("../../memory/segments.zig").MemorySegmentM
 const AutoHashMap = std.AutoHashMap;
 const ArrayList = std.ArrayList;
 const Allocator = std.mem.Allocator;
+const Tuple = std.meta.Tuple;
+
+const EC_POINTS = [_]Tuple(&.{ usize, usize }){
```

Comment:
Why not using an array of `EcPoint` struct directly here? In order to represent the coordinates in a proper way and to avoid the usage of tuples.